exp/api/v1alpha4/azuremanagedmachinepool_types.go

```diff
@@ -56,6 +56,16 @@ type AzureManagedMachinePoolSpec struct {
 	// ProviderIDList is the unique identifier as specified by the cloud provider.
 	// +optional
 	ProviderIDList []string `json:"providerIDList,omitempty"`
+
+	// Scaling specifies the autoscaling parameters for the node pool.
+	// +optional
+	Scaling *ManagedMachinePoolScaling `json:"scaling,omitempty"`
```

Comment:
just noticed this, we shouldn't be adding fields to v1alpha3 and v1alpha4, the main branch targets the v1beta1 release

Reply:
removed them from v1alpha3 and v1alpha4 now.